scripts/migrate_test_runs.py

```diff
@@ -0,0 +1,52 @@
+import sys
+from django.core.exceptions import ObjectDoesNotExist
+from squad.core.models import Project, Build, Environment
+from squad.core.tasks import UpdateProjectStatus
+
+
+def run(*args):
+
+    parsed_args = dict(arg.split("=") for arg in args if '=' in arg)
+
+    if 'old_project_slug' not in parsed_args.keys():
+        print("ERROR: old_project_slug missing")
+        sys.exit(1)
+    if 'new_project_slug' not in parsed_args.keys():
+        print("ERROR: new_project_slug missing")
+        sys.exit(1)
+    if 'env_slug' not in parsed_args.keys():
+        print("ERROR: env_slug missing")
+        sys.exit(1)
+
+    old_project = Project.objects.get(slug=parsed_args['old_project_slug'])
+    new_project = Project.objects.get(slug=parsed_args['new_project_slug'])
+    env = None
+    try:
+        env = Environment.objects.get(project=old_project, slug=parsed_args['env_slug'])
+    except ObjectDoesNotExist:
+        print("Environment: %s not found. Exiting" % parsed_args['env_slug'])
+        sys.exit(0)
+    print("Migrating testruns from project %s to %s" % (old_project.slug, new_project.slug))
+    print("All test runs with environment name: %s will be migrated" % env.slug)
+
+    for build in old_project.builds.all():
+        if build.test_runs.filter(environment=env):
+            print("moving build: %s" % build)
+            new_build, _ = Build.objects.get_or_create(
+                version=build.version,
+                project=new_project,
+                created_at=build.created_at)
+            for testrun in build.test_runs.filter(environment=env):
+                testrun.build = new_build
```

Comment:
do we need to care about the case in which there are no test runs left in the original build?